test/screenshotter/test.html

```diff
@@ -31,7 +31,35 @@
     <span id="math"></span>
     <span id="post"></span>
     <script type="text/javascript">
-      function handle_search_string(search, callback) {
+      function load_fonts(callback) {
+        if (!document.fonts || !document.fonts.load) {
+          callback();
+          return;
+        }
+
+        var results = [];
+        function load_font(name, style) {
+          // 10px is arbitrarily selected value
+          results.push(document.fonts.load((style || "") + " 10px " + name));
+        }
```

Comment:
Neat.  This is pretty straight forward.  If we find this doesn't work when we start testing on more browsers we can have a look at https://github.com/typekit/webfontloader.

Reply:
@kevinbarabash Modern browsers except Edge support this way of loading (https://caniuse.com/#feat=font-loading).

Reply:
Cool.  Hopefully Edge will add this soon. 🤞